src/SbManager/Content/tmpl/directives/error.html

```diff
@@ -0,0 +1,4 @@
+﻿<div class="panel panel-danger">
+    <div class="panel-heading">{{model.Error.Title || "Sorry, something went wrong"}}</div>
+    <div class="panel-body">{{model.Error.Summary || "An unexpected error occurred."}}</div>
```

Comment:
Be nice if we didn't loose context of the error message that would have been displayed on the alert

ie

Reply:
Actually, `error.StatusCode` and `error.Message` do not exist, the error returned on internal server error is [ErrorPageViewModel](https://github.com/GlobalX/SbManager/blob/7aa427324dc9d0119621745ac9b2faa409adac32/src/SbManager/InternalCommandHandlers/ErrorStatusCodeHandler.cs#L48-L53).

Reply:
@0xced another good pick up!